### PR TITLE
Server init

### PR DESCRIPTION
**Pull request by csarkosh**
initial server setup

### CODE REVIEW

File: appspec.yml [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
 version: 0.0
+
 files:
   - source: app.js
     destination: /opt/unicorn
-
   - source: package.json
     destination: /opt/unicorn
+  - source: unicorn.service
+    destination: /opt/unicorn/scripts
```
Comment (dave-symons) [inline appspec.yml:11]: This destination should be `/etc/systemd/system`

File: scripts/after_install.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
 cd /opt/unicorn
-npm install
+sudo npm install
```
Comment (dave-symons) [inline scripts/after_install.sh]: script runs as sudo (look at appspec.yml) so this is not needed

File: scripts/app_start.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+sudo systemctl start unicorn
```
Comment (dave-symons) [inline scripts/app_start.sh]: sudo not needed

File: scripts/before_install.sh [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+sudo systemctl disable unicorn
```
Comment (dave-symons) [inline scripts/before_install.sh]: Sudo not needed, change `disable` to `stop`

File: scripts/unicorn.service [hunk 5]
```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=unicorn
+
+[Service]
+ExecStart=/opt/unicorn/app.js
```
Comment (dave-symons) [inline scripts/unicorn.service]: app.js isn't executable and doesn't start with the node path so probably will have to prefix with `node`
`node /opt/unicorn/app.js`

File: appspec.yml (hunk 1 above)
Comment (csarkosh) [inline appspec.yml:11]: @aosyborg since amazon linux is forked from red hat, I don't believe it has systemd in /etc/

**Pull request closed without merging**